CIFAR10/train/apple_resnet9_8v100_pytorch.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "v1.0",
+    "author": "Santiago Akle Serrano, Hadi Pour Ansari, Vipul Gupta, Dennis DeCoste",
+    "authorEmail": "sakle@apple.com",
+    "framework": "Pytorch 1.1.0",
+    "codeURL": "https://github.com/apple/ml-cifar-10-faster",
```

Comment:
Please specify the commit hash either in the `codeURL` or as a separate `commitHash` field